classes/mail/variables/Variable.inc.php

```diff
@@ -29,7 +29,7 @@ abstract protected static function description() : array;
      * Get the value of variables supported by this class
      * @return string[]
      */
-    abstract protected function values() : array;
+    abstract public function values(string $locale) : array;
```

Comment:
If `Variable::getValue()` isn't used anywhere and there are no such plans, it can be removed.